utils/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "utils"
```

Comment:
`utils` is a bad name. Maybe, `openapi-schema-gen`?

Reply:
There *could* be other utilities in the future, aren't they? It's convenient to keep all kinds of scripts under the same folder